apollo-router/src/layers/mod.rs

```diff
@@ -57,7 +57,7 @@ pub trait ServiceBuilderExt<L>: Sized {
     /// # fn test(service: supergraph::BoxService) {
     /// let _ = ServiceBuilder::new()
     ///     .checkpoint(|req: supergraph::Request|{
-    ///         if req.originating_request.method() == Method::GET {
+    ///         if req.supergraph_request.method() == Method::GET {
```

Comment:
Ah, another benefit of regex over using the language server's "Rename symbol" feature: commented code gets fixed! 😄  (macros are the obvious other one.)